posts/2024-07-08_unix_vs_sas_date.../unix_vs_sas_datetime.qmd

```diff
@@ -0,0 +1,100 @@
+---
+title: "Unix versus SAS Time"
+author:
+  - name: Céline Piraux
+description: "This blog explores SAS and R date handling differences, focusing on epoch discrepancies and data types, and provides key tips for accurate date conversions to prevent a 10-year date shift."
+date: "2024-07-08"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Technical]
+# feel free to change the image
+image: "pharmaverse.PNG"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-07-08_unix_vs_sas_date..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Recent discussions within the R Consortium Submission Working Group have highlighted challenges in handling dates between SAS and Unix systems.
+In addition, during the CDISC Pilot Data update at the Phuse Test Data Factory, using R's Unix time format resulted in dates being mistakenly advanced by +10 years.
```

Comment:
Could we link the Phuse Test Data Factory - https://github.com/phuse-org/TestDataFactory.  Do you mind locating and verifying the time-stamp issue.  https://rconsortium.github.io/submissions-wg/minutes/2024-06-07/  Just want to make sure this is the right one!

Reply:
ACtually the meeting minutes might be nice to link as well!!